src/phoenix/server/api/types/JSONSchema.py

```diff
@@ -1,9 +1,15 @@
 import strawberry
 from strawberry.scalars import JSON
 
+from phoenix.server.api.helpers.prompts.models import PromptJSONSchema
+
 
 @strawberry.type
 class JSONSchema:
     """A JSON schema definition used to guide an LLM's output"""
 
-    schema: JSON
+    definition: JSON
+
+
+def to_gql_json_schema_from_pydantic(pydantic_json_schema: PromptJSONSchema) -> JSONSchema:
```

Comment:
Are we sure we want this to float fully unstructured, or should we store this in the database embedded using our own simple schema like our other types